backend/flask_routes/app.py

```diff
@@ -35,6 +35,12 @@
 from data_handling.data_maps import sum_category_map
 from data_handling.data_processing import create_pie_plot
 
+# notifications imports
```

Comment:
This file is getting real long. You don't have to do it now, but consider in the future to split up to a few files: one for routes, one for cron job, one for helper functions

Reply:
yeah I agree, especially with moving helpers, I'm already doing this a little with the `data_processing.py` file but that's focused on functions that deal with pandas / matplot